src/corona_hakab_model/infection.py

```diff
@@ -63,11 +63,28 @@ def _perform_infection(self) -> Dict[Agent, InfectionInfo]:
 
     def _infect_random_connections(self) -> Dict[Agent, InfectionInfo]:
         connections = self.manager.num_of_random_connections * self.manager.random_connections_factor
-        probs_not_infected_from_connection = np.ones_like(connections, dtype=float)
+
+        probs_not_infected_from_random_connection = self._get_probs_not_infected_from_random_connection(connections)
+        probs_not_infected_from_geo_random_connection = self._get_probs_not_infected_from_geo_random_connection(connections)
+
+        probs_not_infected_from_connection = (probs_not_infected_from_random_connection + probs_not_infected_from_geo_random_connection) - 1
```

Comment:
kinda hacky but ok :laughing:

Reply:
it there a better solution? i couldn't really think of one :sweat_smile: